index.html

```diff
@@ -10,23 +10,246 @@
+        </div>
+        <div class="project-access">See project</div>
+      </div>
+    </div>
+
+    <div class="project" id="project4">
+      <div class="pimage"></div>
+      <div class="projectdetails">
+        <div class="p-title">
+          <h2>Tonic</h2>
+        </div>
+        <ul class="details-list">
+          <li class="client">CANOPY</li>
+          <li class="spacedot"></li>
+          <li class="role">Back End Dev</li>
+          <li class="spacedot"></li>
+          <li class="year">2015</li>
+        </ul>
+        <div class="p-description">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <div class="languages">
+          <ul class="languages-list">
+            <li class="lang">html</li>
+            <li class="lang">css</li>
+            <li class="lang">javaScript</li>
+          </ul>
+        </div>
+        <div class="project-access">See project</div>
+      </div>
+    </div>
+  </section>
+
+  <section class="about-me" id="about">
+    <div class="about-me-content">
+      <div class="about-intro">
+        <div class="primary-text-container">
+          <h2 class="primary-text">About<br>Myself</h2>
+        </div>
+        <div class="secondary-text">
+          <p>I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </div>
+        <h3 class="connect-text">LET'S CONNECT</h3>
+        <div class="socialmedia-icons">
+          <ul>
+            <li class="twitter-icon"><a href=""></a></li>
+            <li class="linkedin-icon"><a href=""></a></li>
+            <li class="m-icon"><a href=""></a></li> 
+            <li class="cat-icon"><a href=""></a></li>
+            <li class="fingers-icon"><a href=""></a></li>
+          </ul>
+        </div>
+        <button class="resume-access">Get my Resume</button>
+      </div>
+      <div class="about-details">
+        <ul class="categories-info">
+          <li class="categorie" id="cat1">
+            <div class="categorie-name">
+              <h4>Languages</h4>
+            </div>
+            <div class="arrow-img"></div>
+          </li>
+          <li class="subcategories-info">
+            <div class="subcategorie" id="subcat1">
+              <div class="subcat-logo"></div>
+              <div class="subcategorie-name">
+                <h4>JavaScript</h4>
+              </div>
+            </div>
+            <div class="subcategorie" id="subcat2">
+              <div class="subcat-logo"></div>
+              <div class="subcategorie-name">
+                <h4>HTML</h4>
+              </div>
+            </div>
+            <div class="subcategorie" id="subcat3">
+              <div class="subcat-logo"></div>
+              <div class="subcategorie-name">
+                <h4>CSS</h4>
+              </div>
```

Comment:
- [ ] [OPTIONAL] Great work so far,  I would like to recommend you to use list `ul` and each technology has a `li` (e.g., "JavasScript", "HTML" and "CSS").